### ISSUE TITLE
Throws undefined errors when using Gatsby

### ISSUE THREAD

**Issue by gigaSproule** [aaronhayes/react-use-hubspot-form#29]
- **I'm submitting a ...**
- [x] bug report
- [ ] feature request
- [ ] question about the decisions made in the repository
- [ ] question about how to use this project

- **Summary**

When using this library with GatsbyJS, depending on what's included in the page, will cause undefined errors to be thrown. The reason for this is because the library loads the script in asynchronously and because it's to the body. This is because with Gatsby, it is rendering the page server side, setting the state as loaded, then when the page is actually loaded into the browser, it is passing the "loaded" check before it's actually been loaded.

If there was an option to load this synchronously and to add the script to the head (as it supposedly should do as per comment https://github.com/aaronhayes/react-use-hubspot-form/blob/master/src/HubspotProvider.tsx#L17), then this will no longer be an issue.

A simple example to replicate this behaviour:
```html
<!DOCTYPE html>
<html lang="en">
<head>
</head>
<body>
<script>
  hbspt.forms.create({
    region: region,
    portalId: portalId,
    formId: formId
  });
</script>
</script>
<script charset="utf-8" type="text/javascript" src="https://js.hsforms.net/forms/v2.js"></script>
</body>
</html>
```

**Comment by aaronhayes** [comment]
Makes sense, happy to accept PRs for this

**Comment by gigaSproule** [comment]
I was planning to do one last night, but didn't get time to. I will try and get one over today.

**Comment by gigaSproule** [comment]
I hope the PR is adequate. I've made it so there are no backwards breaking changes, so existing users won't get any errors when upgrading.

**Comment by aaronhayes** [comment]
Thanks @gigaSproule appreciated it. Released in `2.1.0` by #30
